### PR TITLE
Meeting 2024-01-24

### PR DESCRIPTION
**Pull request by sthagen**
- [x] pre-meeting draft
- [x] post-meeting draft

Ready for review.

### CODE REVIEW

**Review by justmurphy — APPROVED** [review]
Looks good to me! Thank you, Stefan.

**Review by justmurphy — CHANGES_REQUESTED** [review]

File: meeting-minutes/2024-01-24.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+* Thomas Schmidt: Introduces pull request [Core Schema Draft #10](https://github.com/oasis-tcs/openeox/pull/10) focus is on date and status which
+  are missing from CSAF, CycloneDX, and SPDX. So, the Core Schema tries to fill this gap and else profit
+  from existing work. During the meeting Thomas added the shell schema to the PR.
+
+* Omar: Introduces pull request [Initial OpenEoX JSON Schema Draft #8](https://github.com/oasis-tcs/openeox/pull/8) and suggests to document
+  but close with no action with reference to [Core Schema Draft #10](https://github.com/oasis-tcs/openeox/pull/10)
+
+## 3.3 Next steps
+
+* skipped due to elapsed time - everyone invited to chime in per E-Mail to the TC list
```
Comment (tschmidtb51) [inline meeting-minutes/2024-01-24.md:122]:
```suggestion
* skipped due to elapsed time - everyone invited to chime in per email to the TC list

**Review by tschmidtb51 — COMMENTED** [review]
Some comments and clarifications

File: meeting-minutes/2024-01-24.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+| Sonny      | van Lingen | Huawei Technologies Co., Ltd.                               | Voting Member           | Yes     |
+| Stefan     | Arntzen    | Huawei Technologies Co., Ltd.                               | Member                  | Yes     |
+| Stefan     | Hagen      | Individual                                                  | Secretary, taking notes | Yes     |
+| Thomas     | Proell     | Siemens                                                     | Member                  | Yes     |
+| Thomas     | Schaffer   | Cisco Systems                                               | Voting Member           | Yes     |
+| Thomas     | Schmidt    | Federal Office for Information Security (BSI) Germany       | Voting Member           | Yes     |
+| Tobias     | Limmer     | Siemens                                                     | Member                  | No      |
+
+
+Quroum was achieved.
```
Comment (justmurphy) [inline meeting-minutes/2024-01-24.md:36]:
```suggestion
Quorum was achieved.
```

File: meeting-minutes/2024-01-24.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+* PRs on initial schemas and Duncan's scenario directories
+* Next steps
+* Adjourn
+
+The agenda was approved as amended based on the raise of a point of order
+by Jay to add the approval of the previous meeting minutes.
+
+## 1.5 Approval of previous minutes (Co-Chair)
+
+* [Minutes of 2023-12-07 Meeting #1 (Inaugural)](https://www.oasis-open.org/committees/document.php?document_id=71500&wg_abbrev=openeox)
```
Comment (justmurphy) [inline meeting-minutes/2024-01-24.md:57]:
```suggestion
* [Minutes of 2023-11-02 Meeting #1 (Inaugural)](https://www.oasis-open.org/committees/document.php?document_id=71500&wg_abbrev=openeox)
```

File: meeting-minutes/2024-01-24.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+| Altaz      | Valani     | Individual                                                  | Member                  | No      |
+| Denny      | Page       | Individual                                                  | Voting Member           | Yes     |
+| Duncan     | Sparrell   | sFractal Consulting LLC                                     | Voting Member           | No      |
+| Jay        | White      | Microsoft                                                   | Voting Member           | Yes     |
+| Jeremy     | Rickard    | Microsoft                                                   | Voting Member           | Yes     |
+| Justin     | Murphy     | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Co-Chair                | Yes     |
+| Kris       | Vandecruys | Cisco Systems                                               | Voting Member           | Yes     |
+| Kunal      | Modasiya   | Qualys, Inc.                                                | Member                  | No      |
+| Langley    | Rock.      | Dell.                                                       | Voting Member           | No      |
```
Comment (justmurphy) [inline meeting-minutes/2024-01-24.md:20]:
```suggestion
| Langley    | Rock      | Dell                                                       | Voting Member           | No      |
```

File: meeting-minutes/2024-01-24.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+
+Quroum was achieved.
+
+## 1.3 Procedures for this meeting (Co-Chair)
+
+## 1.4 Approval of agenda
+
+* Updates
+  *  CVE/FIRST Vuln Conf and Annual CNA Summit
```
Comment (justmurphy) [inline meeting-minutes/2024-01-24.md:43]:
```suggestion
  *  [CVE/FIRST VulnCon 2024 and Annual CNA Summit](https://www.first.org/conference/vulncon2024/)
```

**Review by justmurphy — COMMENTED** [review]
Added a few more edits and suggestions

**Review by justmurphy — APPROVED** [review]
Looks great!

**Review by justmurphy — APPROVED** [review]
Looks good.

**Pull request merged**